test/functional/page_objects/dashboard_page_controls.ts

```diff
@@ -475,24 +523,43 @@ export class DashboardPageControls extends FtrService {
   }
 
   public async optionsListPopoverSelectExists() {
+    this.log.debug(`select 'Exists/Does not Exist' option in suggestions`);
+    await this.optionsListPopoverAssertOpen();
     await this.retry.try(async () => {
       await this.testSubjects.existOrFail(`optionsList-control-selection-exists`);
       await this.testSubjects.click(`optionsList-control-selection-exists`);
     });
   }
 
-  public async optionsListPopoverSelectOption(availableOption: string) {
-    this.log.debug(`selecting ${availableOption} from options list`);
-    await this.optionsListPopoverSearchForOption(availableOption);
-    await this.optionsListPopoverWaitForLoading();
-
+  public async optionsListPopoverClickShowOnlySelected() {
+    this.log.debug(`click "show only selected" button in options list popover`);
+    await this.optionsListPopoverAssertOpen();
     await this.retry.try(async () => {
-      await this.testSubjects.existOrFail(`optionsList-control-selection-${availableOption}`);
-      await this.testSubjects.click(`optionsList-control-selection-${availableOption}`);
+      await this.testSubjects.existOrFail(`optionsList-control-show-only-selected`);
+      await this.testSubjects.click(`optionsList-control-show-only-selected`);
     });
+  }
 
-    await this.optionsListPopoverClearSearch();
-    await this.optionsListPopoverWaitForLoading();
+  public async optionsListPopoverSelectOptions(availableOptions: string | string[]) {
```

Comment:
Because the logic of checking to ensure the appropriate elements are selected is **pretty** expensive / slow, I decided to reduce the amount we call this by allowing multiple selections to be made at once **before** the check is made; that way, when we previously had multiple consecutive `optionsListPopoverSelectOption` calls, we can now call `optionsListPopoverSelectOptions` with an array and the selections check will only run once.****